example/scripts/sample_data_loader.py

```diff
@@ -37,22 +38,27 @@
 from databuilder.task.task import DefaultTask
 from databuilder.transformer.base_transformer import NoopTransformer
 
-es_host = None
-neo_host = None
+# configured by Kube DNS based on service name
+es_host = "elasticsearch"
+neo_host = "neo4j"
+
+es_port = os.getenv("ELASTICSEARCH_SERVICE_PORT_ELASTICSEARCH_HTTP", 9200)
```

Comment:
how about `CREDENTIALS_ELASTICSEARCH_PROXY_PORT` ?

Reply:
this matches what we called in https://github.com/lyft/amundsensearchlibrary/blob/master/search_service/config.py#L40

Reply:
ah yeah, I didn't even notice that this change was included in the diff - these are environment variables set by Kubernetes, so only really relevant for our specific deployment with the services named the way we have them named.  I will update these, and the hosts above as well, to be more consistent.